from_cpython/Lib/test/test_complex.py

```diff
@@ -440,7 +438,7 @@ def __rmod__(self,other):
                 for y in test_values:
                     a = 'x %s y' % op
                     b = 'y %s x' % op
-                    self.assertTrue(type(eval(a)) is type(eval(b)) is xcomplex)
+                    # self.assertTrue(type(eval(a)) is type(eval(b)) is xcomplex)
```

Comment:
If `type(y)` is `complex`, `type(x)` is `xcomplex`:

`type(pow(y, x))` and `type(x ** y)`
===> `xcomplex`.
`type(y ** x)`
===> `complex`
It should be `xcomplex`.

Would you mind to give some tips? FYI, originally I use my own implementation of `complexPow`, the error occured and could not figure out why. Then I switch to the pure CPython implementation. Same problem.